Package@swift-5.5.swift

```diff
@@ -1,6 +1,6 @@
 // swift-tools-version:5.5
-// In order to support users running on the latest Xcodes, please ensure that
-// Package@swift-5.5.swift is kept in sync with this file.
+// In order to support users running on legacy Xcodes, please ensure that
```

Comment:
Nit: I think "legacy" might not always be appropriate term to use for clients of the `Package.swift` file. I suggest relaxing the phrasing a bit:

```suggestion
// In order to support users running on previous versions of Xcode, please ensure that
```

Reply:
https://github.com/apple/swift-driver/blob/9ef905a68300e82e4dd72fab2ecf369a917b09b1/Package%40swift-5.5.swift#L1-L3

The swift-driver uses "legacy" here. But if you prefer, we can change it.

Reply:
I think "previous versions of Xcode" is more accurate because I think we want to be able to copy/paste this file for future versions of Swift compilers supported by Xcode versions in beta. In those cases, "legacy" wouldn't be appropriate to refer to the version of Xcode that's shipping in the Mac App Store.